### ISSUE TITLE
Wrong indentation in case after brace block

### ISSUE THREAD

**Issue by micheleCTDE** [uncrustify/uncrustify#3037]
uncrustify 0.72.0 + backport of commits 45681e8, 64d93e3.
Files in attachment:

- test.cfg: configuration used for the test
- t1_in.cpp: original test file (subset of a real file, it would fail compilation)
- t1_out_1st_pass.cpp: output of execution

Step to reproduce: 
1. run "uncrustify -c test.cfg -f t1_in.cpp -o t1_out_1st_pass.cpp"

Problem:
indentation of the code after the brace block of the "DCOPReplyDelayed" case if wrong.
```
    case DCOPReplyDelayed:
        {
            ...  (remove for brevity)
        }
            tqWarning("Very strange! got a DCOPReplyDelayed opcode, but we were not waiting for a reply!");
            return;
```

Expected output:
```
    case DCOPReplyDelayed:
        {
            ...  (remove for brevity)
        }
        tqWarning("Very strange! got a DCOPReplyDelayed opcode, but we were not waiting for a reply!");
        return;
```

[wrong_indent_in_switch.zip](https://github.com/uncrustify/uncrustify/files/5744601/wrong_indent_in_switch.zip)

**Comment by guy-maurel** [comment]
you need to use the option:
`indent_case_brace = 2`

**Comment by micheleCTDE** [comment]
Nope, using "indent_case_brace = 2" would result in a different indentation than what we would like to have.

Expected:
```
   case DCOPReplyDelayed:
   {
       [... omissis ]
   }
   tqWarning("Very strange! got a DCOPReplyDelayed opcode, but we were not waiting for a reply!");
   return;
```

With "indent_case_brace = 2"
```
   case DCOPReplyDelayed:
     {
       [... omissis ]
     }
     tqWarning("Very strange! got a DCOPReplyDelayed opcode, but we were not waiting for a reply!");
     return;
```

**Comment by guy-maurel** [comment]
There are two options:
_indent_switch_case_ and _indent_case_brace_
They MUST have the same value to get what you wanted to have!

**Comment by guy-maurel** [comment]
All the cases are pretty well.
Only the case DCOPReplyDelayed: introduces a new block.
Why not!
But why don't you put all the statements in this block?
All the statements (such as _if ( replyStruct ) {_, later _break;_, and _DCOPProcessInternal(d, opc..._)
in the blocks are aligned correctly.
BUT you are using:
  _indent_case_brace = 0_
This is the problem. A solution is shown above.

**Comment by micheleCTDE** [comment]
Hi Guy,
yes, that is an easy way to fix this instance, move the last two statements in 'DCOPReplyDelayed' inside the braces.
Nevertheless when going to use uncrustify on 48k source files, I would much prefer for such cases (if any) to be handled automatically, so I think we could probably improve uncrustify further.

Basically the { } block and the two subsequent instructions are the body of 'case: DCOPReplyDelayed'.
If such block was normal code (non inside a switch statement), people would expect the two instructions to be aligned with the { } block.
So IMO it makes sense that it should be the behavior also within a case block.

This is a low priority issue, but I would like to keep it open for the time being.
